### PR TITLE
refactor(frontend): add fill currentColor to svg phat

### LINKED ISSUE #2524: 🐛 [Bug] Cockpit: Not all Social Media Icons are white in Dark Mode
## 🐛 Bug
Several Social Media Icons are black in dark mode (see screenshots).
They should all be white in dark mode.

cockpit_about_me_categorie_icons_dark_mode_02    cockpit_about_me_categorie_icons_dark_mode

### PR DESCRIPTION
**Pull request by ogerly**
## 🍰 Pullrequest

I have added svgś in to all of them and reduced the displayed size of all of them slightly
```` ... <path fill="currentColor" ....````

### Issues
- fixes #2524 

Bildschirmfoto vom 2024-10-21 09-14-38
Bildschirmfoto vom 2024-10-21 09-14-22

### CODE REVIEW

**Review by Bettelstab — CHANGES_REQUESTED** [review]
The icons are not centered in their viewbox, so they appear slightly off. Please cut the whitespace.

**Comment by ogerly** [comment]
Bildschirmfoto vom 2024-10-23 10-51-01
Bildschirmfoto vom 2024-10-23 10-50-46
Bildschirmfoto vom 2024-10-23 10-04-10
Bildschirmfoto vom 2024-10-23 10-03-59
Bildschirmfoto vom 2024-10-23 10-03-50
Bildschirmfoto vom 2024-10-23 10-03-37

i have replaced the v-img with the svg so that the dark/light mode works.

File: frontend/src/components/cockpit/about-me/AboutMeView.vue [hunk 1]
```diff
@@ -246,4 +246,10 @@ hr {
   margin-block: 10px;
   border-color: white;
 }
+
+.social-icon {
```
Comment (Bettelstab) [inline frontend/src/components/cockpit/about-me/AboutMeView.vue:250]: Class is not longer used, please remove

**Review by Bettelstab — CHANGES_REQUESTED** [review]
Please remove unused code otherwise good to go! Thank you :)

File: frontend/src/components/cockpit/about-me/AboutMeView.vue (hunk 1 above)
Comment (ogerly) [reply frontend/src/components/cockpit/about-me/AboutMeView.vue]: removed

**Review by Bettelstab — APPROVED** [review]

**Pull request merged**